content/en/sensitive_data_scanner/guide/redact_all_emails_except_from_specific_domain_logs.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Redact All Emails Except from a Specific Domain in Logs
+disable_toc: false
+further_reading:
+- link: "/sensitive_data_scanner/"
+  tag: "Documentation"
+  text: "Set up Sensitive Data Scanner"
+---
+
+## Overview
+
+This guide walks you through how to redact all emails, except the ones from a specific email domain (for example, `@test.com`), in your logs.
+
+## Set up a grok processor
+
+If the email domain you do not want redacted is not an existing log attribute, set up a grok processor to identify all logs with the email domain and add it as an attribute.
+
+1. Navigate to [Log Pipeline][1].
+1. Select your pipeline.
+1. Click **Add processor**.
+1. Select **Grok Parser**.
```

Comment:
I see both "grok processor" and "grok parser" used here seemingly to mean the same thing. Should these all be "grok parser"?

Reply:
@neko-dd good catch! It should be "grok parser". I'm getting it mixed up with the grok parser processor in Obs Pipelines 🙃 I fixed it in this [commit](https://github.com/DataDog/documentation/pull/24454/commits/bfb334c5f2b49668123684a2e9c272b67b883539).